src/amo/css/AddonDetail.scss

```diff
@@ -68,7 +68,7 @@
     }
   }
 
-  .about {
+  .about, .overall-rating {
```

Comment:
Generally we put selectors on a new line (maybe stylelint has a rule for this) because if you have a lot of selectors it's quite hard to parse them reading across the page as opposed to vertically.